agent/custom/bbc_connection_manager.py

```
@@ -28,42 +28,95 @@
             'bbc_process': None,  # BBC 进程对象
         }
         self._state_lock = threading.Lock()
         
-        self._initialized = True
-        mfaalog.info("[BbcConnectionManager] 初始化完成")
+        mfaalog.info(f"[BbcConnectionManager] 创建新实例, ID: {id(self)}, Event ID: {id(self._bbc_ready_event)}")
         
         # 自动启动回调监听
         self._start_permanent_listener()
     
+    def _reset_state(self):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_reset_state`

Add return type annotation: `None`

(ANN202)